docs/UserGuide.md

```diff
@@ -3,9 +3,10 @@ layout: page
 title: User Guide
 ---
 
-# User Guide
-
-SoConnect is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, SoConnect can get your contact management tasks done faster than traditional GUI apps.
+SoConnect is a **desktop app for managing contacts and tasks**. It aims to help NUS SoC students stay
```

Comment:
According to the style guide provided, we should not wrap text ourselves.
![image](https://user-images.githubusercontent.com/97268525/197374802-0b14770d-6ecd-4571-bf80-a2a9ed954fbb.png)

Reply:
Oh, I didn't know this, thanks